adcc/testdata/generate_hfdata_ch2nh2_sto3g.py

```diff
@@ -23,7 +23,7 @@
 import sys
 
 from pyscf import gto, scf
-from geometry import xyz
+from .static_data import xyz
```

Comment:
I think this won't work. It has to be an absolute import in these cases.

Reply:
Done.